apps/remix-ide/src/app/plugins/remixd-handle.tsx

```diff
@@ -115,18 +117,10 @@ export class RemixdHandle extends WebsocketPlugin {
           } catch (error) {
             connection(error)
           }
-        },
-        cancelFn: async () => {
-          this.state = State.cancel
-          await this.canceled()
-        },
-        okLabel: 'Connect',
-        cancelLabel: 'Cancel',
-        hideFn: async () => {
-          if (this.state === State.new) await this.canceled()
-        }
       }
-      await this.call('modal', 'modal', mod)
+      else {
```

Comment:
there isn't a `hide` handle?

Reply:
There is no need. If return is undefined it is hide, if false it is cancel but in this case they both cause the plug-in to cancel